x/authz/exported/authorizations.go

```diff
@@ -15,9 +15,22 @@ type Authorization interface {
 
```

Comment:
We should change MethodName to maybe `MessageName` or `MsgTypeURL`

ref: https://github.com/cosmos/cosmos-sdk/issues/9114#issuecomment-823982059

Reply:
We can't simply change to `MsgTypeURL` because it's an attribute a the GenericAuthorization structure. I agree that for interface it will make sense to change that, so I will update the GenericAuthorization structure: https://github.com/cosmos/cosmos-sdk/pull/9042/commits/155913e73d94376701420cd61b80ee82e4e9c322#diff-8cf7d1dbb9bfe6d2ff0e5c61ea6daf61ff35f10fd3e226980f06ad19961a7e58